GDJS/GDJS/Extensions/Builtin/KeyboardExtension.cpp

```diff
@@ -22,6 +22,8 @@ KeyboardExtension::KeyboardExtension() {
       "gdjs.evtTools.input.wasKeyReleased");
   GetAllConditions()["AnyKeyPressed"].SetFunctionName(
       "gdjs.evtTools.input.anyKeyPressed");
+        GetAllConditions()["AnyKeyReleased"].SetFunctionName(
```

Comment:
This is not properly aligned, can you fix the extra spaces?

Reply:
Okay I'm gonna work on it